<launch>
    <node pkg="heading_ctrl" type="heading_ctrl" name="heading_ctrl" output="screen">
        <!-- engine, depth_ctrl connection -->
        <param name="surge_topic" value="/engine/surge" />
        <param name="depth_ctrl_en_srv" value="/depth_ctrl/enable" />
        <param name="depth_sp" value="/depth_ctrl/depth" />

        <!-- sensors -->
        <param name="imu_topic" value="/xsens/imu" />

        <!-- parameters -->
        <param name="gain_heading" value="0.00833" />
    </node>
</launch>
